packages/edge-gateway/src/gateway.js

```diff
@@ -537,7 +537,7 @@ function getTransformedResponseWithCustomHeaders(response) {
 
   clonedResponse.headers.set(
     'content-security-policy',
-    "default-src 'self' 'unsafe-inline' blob: data: ; form-action 'self' ; navigate-to 'self' "
+    "default-src 'self' 'unsafe-inline' 'unsafe-eval' blob: data: ; form-action 'self' ; navigate-to 'self' "
```

Comment:
We have considered this in https://github.com/nftstorage/nftstorage.link/pull/172#discussion_r944408575, but left it out. As it turns out leaving it out prevents WASM module loading which is not a desired behavior.

We could enable just wasm via [unsafe-wasm-eval](https://github.com/WebAssembly/content-security-policy/blob/main/proposals/CSP.md#the-wasm-unsafe-eval-source-directive) directive, however I don't see a good reason to disable js eval so I'm proposing this.